### PR TITLE
Replace F1-optimal threshold strategy with default value

### PR DESCRIPTION
**Pull request by marko-polo-cheno**

### CODE REVIEW

File: kolena/_experimental/object_detection/workflow.py [hunk 1]
```diff
@@ -197,12 +194,12 @@ class ThresholdConfiguration(EvaluatorConfiguration):
     Specify a confidence and IoU threshold to apply to all classes.
     """
 
-    threshold_strategy: Union[Literal["F1-Optimal"], float]
-    """The confidence threshold strategy. It can either be `F1_OPTIMAL="F1-Optimal"` or a fixed confidence threshold
-    such as `0.3` or `0.75`."""
+    threshold_strategy: Union[Literal["F1-optimal"], float] = "F1-optimal"
```
Comment (munkyshi) [inline kolena/_experimental/object_detection/workflow.py]: I gotta say, `F1-optimal` looks odd after seeing so much `F1-Optimal` -- was this discussed already?

**Review by munkyshi — APPROVED** [review]
One point of confusion, but not blocking if this was already discussed.

File: examples/object_detection_2d/object_detection_2d/seed_test_run.py [hunk 2]
```diff
@@ -121,7 +120,6 @@ def setup_evaluator() -> ObjectDetectionEvaluator:
                 min_confidence_score=0.2,
             ),
             ThresholdConfiguration(
-                threshold_strategy=F1_OPTIMAL,
```
Comment (y27choi) [inline examples/object_detection_2d/object_detection_2d/seed_test_run.py:124]: For the example, wouldn't it be better to specify the `threshold_strategy` just to make it explicit?

File: kolena/_experimental/object_detection/workflow.py (hunk 1 above)
Comment (marko-polo-cheno) [reply kolena/_experimental/object_detection/workflow.py]: Title case-ish is fine, I'll change it back 👍

**Pull request merged**